client/lib/sites-list/test/index.js

```diff
@@ -85,6 +85,16 @@ describe( 'SitesList', () => {
 				assert.deepEqual( updatedValue, site[ prop ] );
 			} );
 		} );
+
+		it( 'should not trigger change when sites attributes didn\'t change', () => {
+			const changeCallback = sinon.spy();
+
+			sitesList.once( 'change', changeCallback );
+
+			sitesList.update( cloneDeep( data.original ) );
+
+			assert.isFalse( changeCallback.called );
+		} );
```

Comment:
> Hmm, not sure I understand you 100%. :) I didn't touch the test fixtures (=> test data), only added the above test. It was failing before I removed the is_previewable etc from site schema though.

Thanks I wanted to verify that the new test failed, before we updated code. Sometimes we can accidentally write tests that pass in both situations, which is not useful :)